### PR TITLE
Chores: Implement card adding

### PR DESCRIPTION
**Pull request by SzDabrowski**

### CODE REVIEW

File: src/App.tsx [hunk 1]
```diff
@@ -47,25 +49,41 @@ function App() {
   };
 
   const addCard = () => {
-    const newCard = <NewCard editMode={editMode} setEditMode={setEditMode} />;
+    setDisplayNewCard(true);
+  };
 
-    console.log("added");
+  const saveCard = ( card: any) => {
```
Comment (Sumick) [inline src/App.tsx:55]: **nitpick:** We should not leave any `any` in code. This object structure should be typed

File: src/components/Card/Card.tsx [hunk 2]
```diff
@@ -45,7 +45,7 @@ export const Card = (props: CardI) => {
         <></>
       ) : (
         <div className={styles.corner_wrapper}>
-                <button className={styles.corner_button} onClick={props.setEditMode}>
+                <button className={styles.corner_button} onClick={EditPageClick}>
```
Comment (Sumick) [inline src/components/Card/Card.tsx:48]: **nitpick:** handlers should be named with `camelCase`, not `PascalCase`, so: `onClick={editPageClick}`

**Review by Sumick — CHANGES_REQUESTED** [review]
Looks good! I found two things that we should improve before merging this code

File: src/components/Card/Card.tsx (hunk 2 above)
Comment (SzDabrowski) [reply src/components/Card/Card.tsx]: Should be fixed now

**Comment by SzDabrowski** [comment]
Should be fixed now

**Review by Sumick — APPROVED** [review]
Good job 🙌

**Pull request merged**